web/assets/scss/firm-hub.scss

```diff
@@ -70,3 +70,24 @@ main {
 .govuk-form-group {
   max-width: 40em;
 }
+
+span.rag {
+  font-weight: bold;
+  &.green {
+    color: #28a197 !important;
+  }
+  &.amber {
+    color: #f47738 !important;
+  }
+  &.red {
```

Comment:
This looks like great CSS, one of us should probably have mentioned before you picked up this ticket that we try to use in the inbuilt GDS classes to do stuff like add colours or weight. The idea is that they should be consistent across all the apps. Nick is currently doing some work on colours so I'd have a look at the tags he's used in [here](https://github.com/ministryofjustice/opg-sirius-supervision-workflow/pull/331/files#:~:text=%3Cspan%3E%3Cstrong%20class%3D%22govuk%2Dtag%20govuk%2Dtag%2D%2Dred%22%3EOverdue%3C/strong%3E%3C/span%3E)